adapters/bluesea/bluesea.go

```diff
@@ -0,0 +1,148 @@
+package bluesea
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+
+	"github.com/prebid/openrtb/v19/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type blueseaAdapter struct {
```

Comment:
Please rename this to `adapter`

Reply:
Renamed `adapter` and completed to sync prebid:master